src/LayerSwitcher/LayerSwitcher.tsx

```diff
@@ -132,7 +128,7 @@ export class LayerSwitcher extends React.Component<LayerSwitcherProps, LayerSwit
    */
   cloneLayer = (layer: OlLayerTile | OlLayerGroup) => {
     let layerClone: OlLayerTile | OlLayerGroup;
-    if (layer.getLayers) {
+    if (layer instanceof OlLayerGroup) {
```

Comment:
We had some difficulties with `instanceof` checks. Maybe you leave it as is. Or introduce `isLayerGroup()` to Openlayers. :wink:

Reply:
I introduced a type guard `isLayerGroup` to still benefit from the typings